Makefile

```diff
@@ -0,0 +1,7 @@
+IMAGE=saas
+
+run:
+	docker run --rm -it -p 5227:5227 $(IMAGE)
+
```

Comment:
perhaps better as a run.sh and build.sh? `make` isn't common with nodejs

Reply:
sure, I prefer one file over a bunch of scripts. Together with https://github.com/tj/mmake it documents the project